logtail/logtail.go

```diff
@@ -137,6 +160,10 @@ type Logger struct {
 	metricsDelta   func() string // or nil
 	privateID      PrivateID
 
+	instanceKey             *string
```

Comment:
pointer-to-string to mean optional isn't super common and generally requires docs.  Better to just use the string zero value for unset (but still with little same-line doc like `// or empty if off`)

Reply:
It's a unit32 now, switched to 0 being the off value